content/tutorials/getusermedia/intro/en/index.html

```diff
@@ -471,36 +470,35 @@ <h2 id="toc-webaudio-api">Using getUserMedia with the Web Audio API</h2>
+<p>Since then, driven by the increasingly ubiquitous capabilities of mobile devices, the web has begun to provide much richer functionality. You now have APIs to <a href="https://developers.google.com/web/updates/2016/12/imagecapture" title="Web Fundamentals article about the Image Capture API">take photos and control camera settings</a>, <a href="https://developers.google.com/web/updates/2016/01/mediarecorder" title="Web Fundamentals article: Record Audio and Video with MediaRecorder">record audio and video</a>, and access other types of sensor data, such as location, motion, and device orientation. The <a href="https://developers.google.com/web/updates/2017/09/sensors-for-the-web" title="What is Generic Sensor API?">Generic Sensor framework</a> ties all this together, alongside generic APIs to enable web apps to <a href="https://developers.google.com/web/updates/2016/03/access-usb-devices-on-the-web" title="Access USB Devices on the Web">access USB</a> and <a href="https://developers.google.com/web/updates/2015/07/interact-with-ble-devices-on-the-web" title="Interact with Bluetooth devices on the Web">interact with Bluetooth</a> devices.</p>
 
 <p><code>getUserMedia()</code> was but the first wave of hardware interactivity.</p>
 
 <h2 id="toc-resources">Additional resources</h2>
 
 <ul>
 <li><a href="https://w3c.github.io/mediacapture-main/">W3C specification</a></li>
-<li>Bruce Lawson's <a href="http://html5doctor.com/getusermedia/">HTML5Doctor article</a></li>
-<li>Bruce Lawson's <a href="http://dev.opera.com/articles/view/getusermedia-access-camera-privacy-ui/">dev.opera.com article</a></li>
+<li>Bruce Lawson's <a href="https://html5doctor.com/getusermedia/">HTML5Doctor article</a></li>
+<li>Bruce Lawson's <a href="https://dev.opera.com/articles/view/getusermedia-access-camera-privacy-ui/">dev.opera.com article</a></li>
 <li><a href="https://www.html5rocks.com/en/tutorials/webrtc/basics/#toc-mediastream" title="HTML5 Rocks intro to WebRTC">Getting Started with WebRTC</a></li>
 </ul>
 <h3 id="toc-demos">Demos</h3>
 
 <ul>
-<li><a href="https://webrtc.github.io/samples/" title="WebRTC samples GitHub Pages">WebRTC Samples</a>: canonical demos and code repo</li>
-<li><a href="http://html5-demos.appspot.com/static/getusermedia/photobooth.html">Live Photo booth</a></li>
-<li>Paul Neave's <a href="http://neave.com/webcam/html5/">WebGL Camera Effects</a></li>
-<li><a href="http://html5photobooth.com/">Snapster</a></li>
-<li><a href="http://learningthreejs.com/data/live-video-in-webgl/">Live video in WebGL</a></li>
-<li><a href="http://www.soundstep.com/blog/experiments/jsdetection/">Play Xylophone with your hands</a></li>
+<li><a href="https://webrtc.github.io/samples/" title="WebRTC samples GitHub Pages">WebRTC samples</a>: Canonical demos and code repo</li>
+<li>Paul Neave's <a href="https://neave.com/webcam/html5/">WebGL camera effects</a></li>
+<li><a href="https://html5photobooth.com/">Snapster</a></li>
```

Comment:
Website is down. Use https://github.com/mrdavidburns/HTML5-Photobooth

Reply:
Done.